lib/pbench/server/api/resources/__init__.py

```diff
@@ -1021,8 +1021,13 @@ def _get_dataset_metadata(
             elif i == Dataset.UPLOADED:
                 metadata[i] = UtcTimeHelper(dataset.uploaded).to_iso_string()
             elif Metadata.is_key_path(i, Metadata.USER_METADATA):
+                native_key = Metadata.get_native_key(i)
+                user = None
+                if native_key == Metadata.USER_NATIVE_KEY:
+                    authorized_user: Optional[User] = Auth.token_auth.current_user()
+                    user = authorized_user
```

Comment:
Except the `Optional` here was a type hint (which you dropped) clarifying that `Auth.token_auth.current_user()` could be `None`, which is still true ... or is that why you already resolved the conversation? 😁 

(Then again, it's a bit weird that I type-hinted `authorized_user` here but not `user`... or, in fact, even introduced `authorized_user` in the first place as I just unconditionally copy it to `user` anyway. 😕 Most likely there was some additional logic that got simplified away...)

Reply:
I pre-resolved the comment, in case you got another approval and decided to keep the extra local variables.  🙂

Once I'd done that, I opted to drop the type hint, since it didn't seem to provide much value that wasn't available from context.  🍤